content/docs/guides/_index.md

```diff
@@ -30,7 +27,7 @@ gRPC clients and servers can run and talk to each other in a variety of environm
 
 ### Working with Protocol Buffers
 
-By default, gRPC uses [Protocol Buffers][PB docs], Google’s
+By default, gRPC uses [Protocol Buffers], Google’s
```

Comment:
Nit: I write `[Protocol Buffers][]` (with a training `[]`), this allows some IDEs to give edit-time validation of named markdown links.

Reply:
Oops, forgot about that. Fixed.